cpp/include/Ice/Connection.h

```diff
@@ -118,24 +117,23 @@ namespace Ice
             std::function<void(std::exception_ptr)> exception,
             std::function<void(bool)> sent = nullptr) = 0;
 
-        /// Flush any pending batch requests for this connection. This means all batch requests invoked on fixed proxies
-        /// associated with the connection.
+        /// Flushes any pending batch requests for this connection. This corresponds to all batch requests invoked on
+        /// fixed proxies associated with the connection.
         /// @param compress Specifies whether or not the queued batch requests should be compressed before being sent
         /// over the wire.
-        /// @return The future object for the invocation.
+        /// @return A future that becomes available when the flush completes.
         [[nodiscard]] std::future<void> flushBatchRequestsAsync(CompressBatch compress);
 
-        /// Set a close callback on the connection. The callback is called by the connection when it's closed. The
-        /// callback is called from the Ice thread pool associated with the connection. If the callback needs more
-        /// information about the closure, it can call {@link Connection#throwException}.
+        /// Sets a close callback on the connection. The callback is called by the connection when it's closed. The
+        /// callback is called from the Ice thread pool associated with the connection.
```

Comment:
Does this mean the client thread pool for Outgoing connections, and the server side thread pool for incoming connections?

Reply:
Yes:
 - outgoing connection: client thread pool
 - incoming connection: OA thread pool or server thread pool